test/functions/test_google_maps_function.py

```diff
@@ -62,7 +58,7 @@ def test_get_address_description(mock_get, mock_client):
     mock_client.return_value = mock_instance
 
     # Call the function with a test address
-    result = get_address_description(
+    result = GoogleMapsToolkit().get_address_description(
```

Comment:
Nit: in these tests, can move the instantiation of the Toolkits into a `@pytest.fixture` annotated function.